pypesto/engine/task.py

```diff
@@ -20,7 +20,7 @@ def __init__(self):
         pass
 
     @abc.abstractmethod
-    def execute(self) -> 'OptimizerResult':  # pylint: disable=R0201
+    def execute(self) -> 'OptimizerResult':  # noqa: F821,R0201
```

Comment:
I believe a better way to address is to make sure forward imports are actually well defined. See https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
agreed. is it now like what you meant?

Reply:
yes :)